citi-onboarding/hacka-pride
# Feature/front guests

## What I did
* Changed svg files from `/assets` to their own folders
* Implemented the components `Slider` and `Mentors` and used it on `Guests`
* Made the front-end and requirements to back-end of the section `Guests`

## How to test
1. Open the project and run the command `yarn start` or `npm start` on `/client`
2. At the same time run the command `node .` or `nodemon` on `/server`
3. You can test the `instagram/facebook/linkedIN` icons works with the KeytoneJS and alterate the sections to verify it
4. Test the responsivity with the google tool for developers
